amazon-eks-nodegroup.yaml

```diff
@@ -107,8 +107,28 @@ Parameters:
   NodeAutoScalingGroupMaxSize:
     Type: Number
     Description: Maximum size of Node Group ASG.
+    Default: 4
+
+  NodeAutoScalingGroupDesiredCapacity:
+    Type: Number
+    Description: Desired capacity of Node Group ASG.
     Default: 3
 
+  NodeAutoScalingGroupUpdatePolicyMaxBatchSize:
+    Type: Number
+    Description: MaxBatchSize for the NodeGroup UpdatePolicy
+    Default: 1
+
+  NodeAutoScalingGroupUpdatePolicyMinInstancesInService:
+    Type: Number
+    Description: MinInstancesInService for the NodeGroup UpdatePolicy
+    Default: 1
+
+  NodeAutoScalingGroupUpdatePolicyPauseTime:
+    Type: String
+    Description: PauseTime for the NodeGroup UpdatePolicy
+    Default: "PT0S"
```

Comment:
What is PT0S?  Do we need some documentation on what values are accepted here?

Reply:
It is the default, zero value. Added a doc url.